frontend/src/contexts/HostContext/hostReducerMiddleware.js

```diff
@@ -21,6 +21,17 @@ function sendPlayerConnectedMessages(payload) {
   });
 }
 
+function sendNameTakenMessage(payload) {
+  socketInstance.sendMessage({
+    event: 'update',
+    recipients: [payload.playerId],
+    payload: {
+      gameState: 'connection-refused-name-taken',
+      removeLoading: 'joining-lobby',
+    },
+  });
+}
+
```

Comment:
In the future maybe we could make this message appear in a modal so the user can just tap out of it and adjust their name without needing to click Join and enter the code again